WeakAuras/SubRegionTypes/Background.lua

```diff
@@ -4,11 +4,7 @@ local L = WeakAuras.L;
 
 do
     local function subSupports(regionType)
-        return regionType == "texture"
-        or regionType == "progresstexture"
-        or regionType == "icon"
-        or regionType == "aurabar"
-        or regionType == "text"
+        return not regionType:match("group")
```

Comment:
I kinda perfer being explicit, that is checking against "group" and "dynamicgroup". That makes it easier to find the code if e.g. searching for everything dynamic group related.